content/questions/admin/question-007.md

```diff
@@ -0,0 +1,16 @@
+---
+question: "Which endspoints does the GitHub API provide to administer your enterprise?"
+archetype: "questions"
+title: "Question 007"
+draft: false
+---
+
+> https://docs.github.com/en/enterprise-cloud@latest/rest/enterprise-admin?apiVersion=2022-11-28
+- [x] Admin stats
+- [x] Audit log
+- [x] Billing
+- [x] Code security and analysis
+- [x] License
+- [x] SCIM
+- [ ] Marketplace
+- [ ] Actions
```

Comment:
Even though it's not in the docs some could argue that Actions API has some impact on administering an enterprise.
For example https://docs.github.com/en/enterprise-cloud@latest/rest/actions/self-hosted-runners?apiVersion=2022-11-28#create-a-registration-token-for-an-enterprise

So let's change this to something else
```suggestion
- [ ] Projects
```